### DISCUSSION TITLE
Connection to an external tenant

### DISCUSSION THREAD

**Discussion by Gm369deg** [cricketthomas/AzureKeyVaultExplorer#90] [General]
Hi, is there a way to open a key vault from an different tenant as a guest user with appropriate permissions ?

**Comment by cricketthomas** [comment]
There is not, currently. It may be possible if the token retrieved for the user is valid on that tenant, but i would have to develop a POC and test it. 

Can you open an issue? If you can just describe how you'd want the feature to work I can try to make something happen. 

My idea for this would possibly just a popup window with the option to enter the fully qualified url of the vault, and it would just open in another tab.

**Comment by cricketthomas** [comment]
> Hi, is there a way to open a key vault from an different tenant as a guest user with appropriate permissions ?
> 
> 

Working on a solution to this. Will be available soon!

See #147
